### PR TITLE
LRQA-79872

### PR DESCRIPTION
**Pull request by Kvale1733**
https://liferay.atlassian.net/browse/LRQA-79872

Resent from https://github.com/liferay-core-infra/liferay-portal/pull/5506

Just modified the test the use the current properties for not running a test in a virtual instance. Since I was still using the previous one which required the entire test name. Everything else is the same.

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:relevant

**Review by yuhai — APPROVED** [review]
@dantewang, compare with https://github.com/liferay-core-infra/liferay-portal/pull/5506, the difference is https://github.com/liferay-core-infra/liferay-portal/blob/74864b1b85a929c9f1b37e5f933e3e82f3948e75/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/beanportlet/BeanPortlet.testcase#L32-L33

I didn't review the testcase part. Based on the comment from https://github.com/liferay-core-infra/liferay-portal/pull/5417#issuecomment-1710241483, I think we may forward it.

cc @Kvale1733

File: modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/src/main/webapp/WEB-INF/liferay-portlet.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!DOCTYPE liferay-portlet-app PUBLIC "-//Liferay//DTD Portlet Application 7.4.0//EN" "http://www.liferay.com/dtd/liferay-portlet-app_7_4_0.dtd">
+
+<liferay-portlet-app>
+	<portlet>
+		<portlet-name>1</portlet-name>
```
Comment (dantewang) [inline modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/src/main/webapp/WEB-INF/liferay-portlet.xml:6]: @Kvale1733 Can we use a more meaningful name?

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/src/main/webapp/WEB-INF/liferay-portlet.xml (hunk 1 above)
Comment (Kvale1733) [reply modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/src/main/webapp/WEB-INF/liferay-portlet.xml:6]: I'll change the name

**Comment by Kvale1733** [comment]
ci:close

**Comment by Kvale1733** [comment]
ci:reopen

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@Kvale1733 Looks good to me then.

**Comment by Kvale1733** [comment]
@dantewang Can we forward this since there is no other issue? Thanks.

**Comment by dantewang** [comment]
@Kvale1733 No other issue from my side. Not sure if it needs further review from QA team... But I think it's OK to go to the next stage.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/141249

**Pull request closed without merging**